tokio-rustls/src/common/mod.rs

```diff
@@ -343,5 +326,24 @@ where
     }
 }
 
+/// An adaptor that implements a [`Read`] interface for [`AsyncRead`] types and an
+/// associated [`Context`].
+pub struct Reader<'a, 'b, T> {
```

Comment:
Maybe we could give this a more descriptive name now that it's used in other places as well? Maybe something like `SyncReadAdapter`? (It appears `adapter` is a more common spelling than `adaptor`.)

Reply:
Great idea, done :+1: